recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+go_compiler:
+  - go-nocgo
```

Comment:
I expect this to be redundant when I finish the work on conda-forge/conda-forge-pinning-feedstock#169.